docs/reference/api/connectivity/bluetooth/bluetooth.md

```diff
@@ -1,32 +1,57 @@
 
-    - [LightBlue](https://itunes.apple.com/gb/app/lightblue-bluetooth-low-energy/id557428110?mt=8) for iPhone.
+The method used to provide the link is called a beacon. You can attach beacons to anything you want to provide information about or that you can provide any sort of interface for.
 
-1. Compile and run our BLE samples:
+For example, you can attach the beacon to a vending machine. It sends you to a web interface that gives you control of the machine. It can let you make a large purchase (providing sodas for several people in one transaction) by letting you select several options and pay for them all at once.
 
-    - **Arm Mbed OS 5 samples** are available on [os.mbed.com](https://os.mbed.com/teams/mbed-os-examples/) and [GitHub](https://github.com/ARMmbed/mbed-os-example-ble):
-        - The [beacon](https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-ble-Beacon/) example is a good starting point; it demonstrates how you can create a BLE beacon with just a few lines of code.  
-        - The [heart rate monitor](https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-ble-HeartRate/) example demonstrates how to build a heart rate sensor that can be connected and monitored by a BLE client such as your phone.
-        - The [LED](https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-ble-LED/) and [LED blinker](https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-ble-LEDBlinker/) are a single example, which demonstrates how a client (LED) and a server (LED blinker) work together over BLE.
+### How a BLE device gets internet access
 
-    <span>**Tip:** Despite the differences between the different Mbed OS versions, there is only **one** version of Mbed BLE, and it is easy to move code from one version of the OS to another. Choose the sample you use according to the version of Mbed OS supported by your development board.</span>
+At the moment, BLE devices don't have independent internet access. To get internet access, you can do one of two things:
 
-#### Going further
+1. You can give your board a secondary communication method, such as Ethernet or Wi-Fi. This can easily double the price of the board.
 
-- The Mbed BLE [API reference](/docs/development/reference/ble.html) offers the full details of the API.
+1. The BLE device can get internet over its BLE connection to a mobile phone. This means that when the phone terminates the BLE connection, the BLE device loses its internet access. This doesn't require additional hardware, so it doesn't affect the price of the board. However, for the device to have constant internet access, it needs a phone (or BLE-enabled computer) next to it.
```

Comment:
"The BLE device can get internet over its BLE connection to a mobile phone." --> "The BLE device can use a gateway (which can be a mobile phone) to exchange data with the internet. The gateway will need to adapt the protocols used as BLE is a non-IP technology."